packages/client/src/components/Modals/SearchPanel.vue

```diff
@@ -218,6 +220,9 @@ export default {
         this.$refs.searchPanelSideBar.hide();
       }
     },
+    isEditMode() {
+      this.initFormState();
+    },
```

Comment:
This is to force the page to reset form values when switching between edit/view mode. This shouldn't actually happen with the backdrop, but leaving in since it won't hurt anything and would be needed if the backdrop were removed.